components/brave_rewards/browser/rewards_protocol_navigation_throttle_unittest.cc

```diff
@@ -26,18 +27,18 @@ bool IsValidWalletProviderRedirect(
     const std::map<std::string, std::vector<GURL>>& allowed_referrer_urls);
 
 // clang-format off
-using RewardsProtocolHandlerTestParamType = std::tuple<
+using RewardsProtocolNavigationThrottleTestParamType = std::tuple<
     std::string,  // test name suffix
     std::string,  // referrer URL
     std::string,  // redirect URL
     bool          // expected result
 >;
 // clang-format on
 
-class RewardsProtocolHandlerTest
-    : public TestWithParam<RewardsProtocolHandlerTestParamType> {};
+class RewardsProtocolNavigationThrottleTest
```

Comment:
we also need a browser test for this that works on desktop and android or we have no way to verify that this is actually working